api.md

```diff
@@ -2029,51 +2109,57 @@ background worker checks the policy. If you need to change the data retention
 interval or reorder by a different index, you'll need to remove the policy and
 add a new one.
 
-#### Required Arguments [](alter_job_schedule-required-arguments)
+#### Required Arguments [](alter_job-required-arguments)
 
 |Name|Description|
 |---|---|
 | `job_id` | (INTEGER) the id of the policy job being modified |
 
 
-#### Optional Arguments [](alter_job_schedule-optional-arguments)
+#### Optional Arguments [](alter_job-optional-arguments)
 
 |Name|Description|
 |---|---|
 | `schedule_interval` | (INTERVAL)  The interval at which the job runs |
 | `max_runtime` | (INTERVAL) The maximum amount of time the job will be allowed to run by the background worker scheduler before it is stopped |
 | `max_retries` | (INTEGER)  The number of times the job will be retried should it fail |
 | `retry_period` | (INTERVAL) The amount of time the scheduler will wait between retries of the job on failure |
-| `if_exists` | (BOOLEAN)  Set to true to avoid throwing an error if the job does not exist, a notice will be issued instead. Defaults to false. |
+| `scheduled` | (BOOLEAN)  Set to `FALSE` to exclude this job from being run as background job. |
+| `config` | (JSONB) Job-specific configuration (this will be passed to the function when executed)|
 | `next_start` | (TIMESTAMPTZ) The next time at which to run the job. The job can be paused by setting this value to 'infinity' (and restarted with a value of now()). |
+| `if_exists` | (BOOLEAN)  Set to true to avoid throwing an error if the job does not exist, a notice will be issued instead. Defaults to false. |
 
-#### Returns [](alter_job_schedule-returns)
+#### Returns [](alter_job-returns)
 
 |Column|Description|
 |---|---|
+| `job_id` | (INTEGER) the id of the policy job being modified |
 | `schedule_interval` | (INTERVAL)  The interval at which the job runs |
 | `max_runtime` | (INTERVAL) The maximum amount of time the job will be allowed to run by the background worker scheduler before it is stopped |
 | `max_retries` | (INTEGER)  The number of times the job will be retried should it fail |
 | `retry_period` | (INTERVAL) The amount of time the scheduler will wait between retries of the job on failure |
+| `scheduled` | (BOOLEAN)  True if this job will be executed by the TimescaleDB scheduler. |
+| `config` | (JSONB) Job-specific configuration (this will be passed to the function when executed)|
+| `next_start` | (TIMESTAMPTZ) The next time at which to run the job. |
 
-#### Sample Usage [](alter_job_schedule-examples)
+#### Sample Usage [](alter_job-examples)
 
 ```sql
-SELECT alter_job_schedule(job_id, schedule_interval => INTERVAL '2 days')
+SELECT alter_job(job_id, schedule_interval => INTERVAL '2 days')
 FROM timescaledb_information.reorder_policies
 WHERE hypertable = 'conditions'::regclass;
```

Comment:
Would this query work without the explicit `::regclass` cast/specification in the WHERE clause?  I realize this is correct / "more" correct, but the typing gets slightly more confusing for your average user.  So perhaps would be preferable to show examples without it, _assuming_ those work.

Reply:
no